lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -309,8 +390,10 @@ class MongoClientInterface {
         m.update({
             _id: bucketName,
         }, {
-            _id: bucketName,
-            value: newBucketMD,
+            $set: {
```

Comment:
what 's the difference (with vs w/o `$set`) ?

Reply:
without $set the vFormat gets removed